tests/GraphQl/Resolver/Stage/SecurityStageTest.php

```diff
@@ -96,4 +96,18 @@ public function testNotGranted(): void
             'extra_variables' => $extraVariables,
         ]);
     }
+
+    public function testNoSecurityBundleInstalled(): void
+    {
+        $this->securityStage = new SecurityStage($this->resourceMetadataFactoryProphecy->reveal(), null);
+
+        $operationName = 'item_query';
+        $resourceClass = 'myResource';
+        $isGranted = 'not_granted';
+        $extraVariables = ['extra' => false];
+        $resourceMetadata = (new ResourceMetadata())->withGraphql([$operationName => ['security' => $isGranted]]);
+        $this->resourceMetadataFactoryProphecy->create($resourceClass)->willReturn($resourceMetadata);
+        $this->expectException(\LogicException::class);
+        ($this->securityStage)($resourceClass, 'item_query', ['extra_variables' => $extraVariables]);
```

Comment:
Why the CS is so different (new lines, ...)?

Reply:
That was maybe my IDE setup for CS, I will fix it in the next commit :)